tests/Unit/Countries/US/NorthCarolina/NorthCarolinaIncome/V20190101/NorthCarolinaIncomeTest.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\NorthCarolina\NorthCarolinaIncome;
+
+use Appleton\Taxes\Models\Countries\US\NorthCarolina\NorthCarolinaIncomeTaxInformation;
+use Carbon\Carbon;
+
+class NorthCarolinaIncomeTest extends \TestCase
+{
+    public function setUp()
+    {
+        parent::setUp();
+
+        Carbon::setTestNow(
+            Carbon::parse('January 1, 2018 8am', 'America/Chicago')->setTimezone('UTC')
+        );
+    }
+
+    public function testCaseStudy2019A()
+    {
+        NorthCarolinaIncomeTaxInformation::forUser($this->user)->update([
+            'additional_withholding' => 0,
+            'dependents' => 0,
+            'filing_status' => NorthCarolinaIncome::FILING_SINGLE,
+        ]);
+
+        $results = $this->taxes->calculate(function ($taxes) {
+            $taxes->setHomeLocation($this->getLocation('us.north_carolina'));
+            $taxes->setWorkLocation($this->getLocation('us.north_carolina'));
+            $taxes->setUser($this->user);
+            $taxes->setEarnings(270.00);
+            $taxes->setPayPeriods(52);
+            $taxes->setYtdEarnings(7340.5);
+            $taxes->setDate($this->date('2019-01-01'));
+        });
+
+        $this->assertEquals(4.00, $results->getTax(NorthCarolinaIncome::class));
+    }
+
+    public function testCaseStudy2019B()
+    {
+        NorthCarolinaIncomeTaxInformation::forUser($this->user)->update([
+            'additional_withholding' => 0,
+            'dependents' => 3,
+            'filing_status' => NorthCarolinaIncome::FILING_SINGLE,
+        ]);
+
+        $results = $this->taxes->calculate(function ($taxes) {
+            $taxes->setHomeLocation($this->getLocation('us.north_carolina'));
+            $taxes->setWorkLocation($this->getLocation('us.north_carolina'));
+            $taxes->setUser($this->user);
+            $taxes->setEarnings(785.00);
+            $taxes->setPayPeriods(52);
+            $taxes->setYtdEarnings(17845);
+            $taxes->setDate($this->date('2019-01-01'));
+        });
+
+        // spreadsheet has 24
```

Comment:
do we need to capture that these values are different than expected on the speadsheet?